model_zoo/vlm/intern_vl/model_adapter_v2_5.py

```diff
@@ -158,8 +160,6 @@ def __getitem__(self, index):
         pixel_values = []
         num_patches_list = []
         for img_path in img_paths:
-            if "dummy" in img_path:
```

Comment:
What was the original purpose of these two lines of code?

Reply:
For older versions of the dataset, I added a dummy image when questions had no associated images. However, some models like InternVL support text-only input, so I skipped this step for those cases. In the latest dataset version, I removed the dummy images entirely. The decision to include dummy images is now determined by the model_adapter.